client/public/index.html

```diff
@@ -11,6 +11,7 @@
     -->
   <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
   <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700|Source+Code+Pro:400,700&amp;subset=latin-ext" rel="stylesheet">
```

Comment:
I think it's better to have it in ```custom.scss``` so that we have css only in scss files. YOu can do:
@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700|Source+Code+Pro:400,700&amp;subset=latin-ext.css')

Reply:
I don't agree with this one. More info here: [Including Google Web Fonts link or import?](https://stackoverflow.com/questions/12316501/including-google-web-fonts-link-or-import)

Reply:
Oh, I see. Didn't know about this :)